pkg/apps/index.html

```diff
@@ -24,7 +24,8 @@
     <meta charset="utf-8">
     <link href="apps.css" type="text/css" rel="stylesheet">
     <script type="text/javascript" src="../base1/cockpit.js"></script>
-    <script type="text/javascript" src="../*/po.js"></script>
+    <script type="text/javascript" src="../base1/po.js"></script>
```

Comment:
You made a good point in https://github.com/cockpit-project/cockpit/pull/16982/commits/b2453bab40a8ad3fdb87f2dcd9929c9fad9388e4 -- maybe we should keep the current cockpit-po-plugin hack that includes all base1 translations into each .po file, and drop this separate inclusion. That means slightly bigger files, but saves the extra roundtrip for this additional file. Especially as we don't cache it any more?

Reply:
I disagree with this.

 - these files get loaded in parallel, so even if base1/po.js isn't cached, it's not going to hurt us much
 - the real fix for this problem is to fix caching, which we can do by making language switching more explicit, if that's what we want

Reply:
Ack.

Reply:
One more point: this solution is only available to modules which live in the `cockpit` source tree.  External modules will have to include `base1/` separately, as there's no way for them to inspect the .po files for `base1/` at their build time.

The other way to fix this, if you really hate the separate include, is as was suggested in the original issue: integrate the po into base1 itself.  But that means we'd need to pick two options:
 - have per-language variants of `cockpit.js`, which, after the experiences of the last few days seems like a very very bad idea
 - include translations for all languages

I don't consider either of those options to be good.